src/modules/Recipes.js

```diff
@@ -16,6 +16,19 @@ const Recipes = {
     } catch (error) {
       return error;
     }
+  },
+  async create(recipe) {
+    try {
+      const response = await api.post("/recipes", {
+        recipe: {
+          name: recipe.name,
+          instructions: recipe.instructions
+        }
+      });
+      return response;
```

Comment:
You want to return the "response.data" here. It's more manageable in the RecipeCreateForm.